packages/gatsby-plugin-ux-capture/.gitignore

```diff
@@ -0,0 +1,6 @@
+/*.js
+!index.js
+yarn.lock
```

Comment:
Why ignore the yarn.lock?

Reply:
Probably a default from plugin repo I copied. In this case though, yarn.lock is in the root of the monorepo anyway.

Reply:
I'll remove it.